change/@office-iss-react-native-win32-2021-01-11-13-17-22-sammy-0.62_focusmanager_backport.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
This needs to be changed to "patch" instead of "none" to generate a new release

Reply:
Thanks, just changed